src/docs/product/discover-queries/index.mdx

```diff
@@ -72,19 +72,17 @@ To rename a saved query, click on the pencil icon by the header and enter the de
 
 ### Share Queries
 
-Share your queries as often as you want. You can share URLs with other users who also have access to the same organization. As each part of the query is built, the results update, and the URL is updated so that in-progress searches can be shared in email, chat, etc.
+Share your queries as often as you want. You can share URLs with other users who also have access to the same organization. As each part of the query is built, the results update and the URL is updated so that in-progress searches can be shared in an email, chat, etc.
```

Comment:
I see this was in the original - but we don't use Latin abbreviations. Correcting now :) 
```suggestion
Share your queries as often as you want. You can share URLs with other users who also have access to the same organization. As each part of the query is built, the results update and the URL is updated so that in-progress searches can be shared in an email, chat, and so forth.
```